examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
```

Comment:
We we need an AppDelegate since we have a SceneDelegate, which is the key SwiftUI concept?

Reply:
This was automatically generated. Yes I think you are right we just need the scene